snappy/config.go

```diff
@@ -61,6 +61,7 @@ func snapConfig(snapDir string, rawConfig []byte) (newConfig []byte, err error)
 		return nil, ErrPackageNotFound
 	}
 
+	// XXX: new security will not make this anymore!!
```

Comment:
Suspect we'll kill config on Monday, and then reintroduce after release via the manager.